### PR TITLE
BOCM-301 - add inspector issue decision endpoint

### PR DESCRIPTION
**Pull request by brpins**
## Issue ticket number and link
https://github.com/Planning-Inspectorate/back-office/tree/BOCM-301

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/src/server/app/inspector/__tests__/issue-decision.test.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+	t.is(response.status, 403);
+	t.deepEqual(response.body, {
+		errors: {
+			userid: 'User is not permitted to perform this action.'
+		}
+	});
+});
+
+test('fails with a 400 status when an invalid `outcome` is present', async (t) => {
```
Comment (mayya-bond-kc) [inline apps/api/src/server/app/inspector/__tests__/issue-decision.test.js:87]: Could we also add a test case when someone tries to call this endpoint to make a decision for an appeal that is not in the "decision_due" status?

Comment (brpins) [reply apps/api/src/server/app/inspector/__tests__/issue-decision.test.js:105]: Added!

File: apps/api/src/server/app/inspector/inspector.validators.js [hunk 2]
```diff
@@ -77,13 +89,31 @@ export const validateBookSiteVisit = composeMiddleware(
 	handleValidationError
 );
 
+// TODO: are there any validation rules on this decision letter upload? Added
```
[the remark this answers is not in the document]
Comment (brpins) [reply apps/api/src/server/app/inspector/inspector.validators.js:92]: Yeah, the thing is that, unlike other pages that restrict the file types, the UI for decision letter doesn't mention these restrictions. Maybe that's an oversight in the UI from the designers, not sure.

**Review by mayya-bond-kc — APPROVED** [review]

**Review by radum — APPROVED** [review]

**Pull request merged**